platform/docs/docs/faq.md

```diff
@@ -29,6 +29,9 @@ If you have resources and would like to fund the development of a feature,
 please [contact us](https://www.ohif.org) or work with community members that
 offer [consulting services][commercial-support].
 
+### Cross Origin Isolation
```

Comment:
Since this is an FAQ, consider using a question instead. Perhaps, "Why do I keep seeing a Cross Origin Isolation warning?"